src/components/shared/Entity/CatalogEditor.tsx

```diff
@@ -15,9 +20,10 @@ function CatalogEditor({ messageId }: Props) {
 
     const draftId = useEditorStore_id();
 
+    const formStatus = useFormStateStore_status();
     const formActive = useFormStateStore_isActive();
 
-    if (draftId) {
+    if (draftId && formStatus !== FormStatus.INIT) {
```

Comment:
We do not want to show the catalog editor unless the user has done something on the forms. 

Otherwise an empty editor showed up right away.